### PR TITLE
Refactor carto values and secrets

### PR DESCRIPTION
**Pull request by alvarorm22**
Refactor carto values and custom client values

Include template for google service account

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

### CODE REVIEW

**Comment by alvarorm22** [comment]
https://app.shortcut.com/cartoteam/story/204140/helm-chart-configuration-and-customer-package

File: chart/values.yaml [hunk 1]
```diff
@@ -8,21 +18,14 @@ cartoConfigValues:
...
-  workspaceThumbnailsBucket: ""
-  workspaceThumbnailsPublic: ""
   reactAppLaunchDarklyClientSideId: ""
-  gcsBucketsProjectId: ""
 
-cartoSecrets:
+clientSecrets:
   googleMapsApiKey: ""
+
+cartoSecrets:
```
Comment (ManuelLR) [inline chart/values.yaml:28]: I think that the name should be:
```suggestion
cartoSecretsValues:
```

File: chart/values.yaml [hunk 2]
```diff
@@ -8,21 +18,14 @@ cartoConfigValues:
...
   selfHostedGcpProjectId: ""
   selfHostedTenantId: ""
-  workspaceImportsBucket: ""
-  workspaceThumbnailsBucket: ""
-  workspaceThumbnailsPublic: ""
   reactAppLaunchDarklyClientSideId: ""
-  gcsBucketsProjectId: ""
 
-cartoSecrets:
+clientSecrets:
```
Comment (ManuelLR) [inline chart/values.yaml:25]: I think that the name should be:
```suggestion
customSecretsValues:
```

**Review by ManuelLR — COMMENTED** [review]
Maybe we should rename to use the same names for secrets?

**Pull request merged**